### DISCUSSION TITLE
New feature suggested - List of tasks for pomodoro

### DISCUSSION THREAD

**Discussion by azimmermanm** [Splode/pomotroid#178] [Q&A]
I would love to have a place where to put list of to do. Then estimate the Pomodoro's per task, and then be able if my estimates were right. 

Obviously I want to say that it's already a great app and I use it every day.
Thanks!

**Comment by y-martini** [comment]
@azimmermanm, should it be in [Ideas](https://github.com/Splode/pomotroid/discussions/categories/ideas) category?

**Comment by blackslashcreative** [comment]
would love this. I've been looking for years for a good Pomodoro timer with modern minimal design, always on top mode, and ability to put in a task / title name for the current session to keep you focused on that. Like most, this one checks 2 of the 3 boxes! :)
